### ISSUE TITLE
Accordion headers don't have `role="button"`

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus-bootstrap.trimble.com#154]
Clicking on accordion header expands/collapses so it should have `role="button"`. It currently fails Lighthouse accessibility test due to this missing.

![image](https://user-images.githubusercontent.com/1212885/177999835-f7200e0b-636a-4f50-b1ab-e599a13a4f07.png)

### What operating system(s) are you seeing the problem on?

Windows
